prototype/KeyGuardMaa/AttestationClient.cs

```diff
@@ -2,27 +2,33 @@
 // Licensed under the MIT License.
 
 using Microsoft.Win32.SafeHandles;
+using System;
 using System.Runtime.InteropServices;
 using static KeyGuard.Attestation.AttestationErrors;
 
 namespace KeyGuard.Attestation
 {
     /// <summary>
-    /// Managed facade for <c>AttestationClientLib.dll</c>.  Holds initialization state,
+    /// Managed façade for <c>AttestationClientLib.dll</c>.  Holds initialisation state,
```

Comment:
What are these changes? Can you undo? The second one is misspelled now.

Reply:
I need to find a way to disable my local spell checker

changes are here - https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5339/commits/3cb79e2805d103ad4d4ed3d0d6f3c121bf8401aa